can_tools/scrapers/official/AZ/AZ_TableauPull.py

```diff
@@ -0,0 +1,280 @@
+                # Get county data
+                cy = extractData(fdat, county[1])
+                outDf = outDf.append(cy, ignore_index=True)
+
+        outDf["CumPosTests"] = outDf["PercentPositive"] * outDf["Number of tests"]
+        outDf["CumDiagPosTests"] = (
+            outDf["Percent Positive Diagnostic tests"]
+            * outDf["Number of Diagnostic tests"]
+        )
+
+        # NOTE: There is currently a bug in the AZDHS dashboard summary page. They do NOT show correct values for antibody positivity rate
+        outDf["CumSeroPosTests"] = outDf["CumPosTests"] - outDf["CumDiagPosTests"]
+        if self.location_type == "county":
+            outDf = outDf[outDf["location"] != 4]
+        return outDf
+
+    def normalize(self, data):
+        df = data
+        crename = {
+            "New Cases": CMU(category="cases", measurement="new", unit="people"),
+            "New Deaths": CMU(category="deaths", measurement="new", unit="people"),
+            "New Tested": CMU(
+                category="unspecified_tests_total", measurement="new", unit="specimens"
+            ),
+            "New Tested PCR": CMU(
+                category="antigen_pcr_tests_total", measurement="new", unit="specimens"
+            ),
+            "New Tested serology": CMU(
+                category="antibody_tests_total", measurement="new", unit="specimens"
+            ),
+            "Number of Cases": CMU(
+                category="cases", measurement="cumulative", unit="people"
+            ),
+            "Number of Diagnostic tests": CMU(
+                category="antigen_pcr_tests_total",
+                measurement="cumulative",
+                unit="specimens",
+            ),
+            "Number of deaths": CMU(
+                category="deaths", measurement="cumulative", unit="people"
+            ),
+            "Number of tests": CMU(
+                category="unspecified_tests_total",
+                measurement="cumulative",
+                unit="specimens",
+            ),
+            "Number of tests serology": CMU(
+                category="antibody_tests_total",
+                measurement="cumulative",
+                unit="specimens",
+            ),
+            "CumPosTests": CMU(
+                category="unspecified_tests_positive",
+                measurement="cumulative",
+                unit="specimens",
+            ),
+            "CumDiagPosTests": CMU(
+                category="antigen_pcr_tests_positive",
+                measurement="cumulative",
+                unit="specimens",
+            ),
+            "CumSeroPosTests": CMU(
+                category="antibody_tests_positive",
+                measurement="cumulative",
+                unit="specimens",
+            ),
+        }
+
+        out = (
+            df.melt(id_vars=["location"], value_vars=crename.keys())
+            .assign(
+                dt=self._retrieve_dt("US/Eastern"), vintage=self._retrieve_vintage()
```

Comment:
Arizona should use ` US/Arizona` as the timezone in `self._retrieve_dt` method (we must not have told you that it should refer to a location's local time)

Reply:
Ah, silly error on my part, I thought maybe this was a reference to when the query is performed from a machine on the east coast.  Thank you!